procrastinate/contrib/django/utils.py

```diff
@@ -36,3 +37,11 @@ def get_setting(name: str, *, default) -> Any:
 
 def package_is_installed(name: str) -> bool:
     return bool(importlib.util.find_spec(name))
+
+
+def package_is_version(name: str, version: int) -> bool:
+    """Assumes the version is written with dots"""
+    v = metadata.version(name)
+    if "." not in v:
+        return False
+    return int(v.split(".")[0]) == version
```

Comment:
`split` will always give you at least one element, so we can remove the `if` clause.
Also, there's less risk of strange exception by comparing  `v.split(".")[0] == str(version)`, I guess (though it should happen on psycopg).
Finally, I think I prefer `version` as a variable name to `v`.